package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "metalsmith-webpack-dev-server",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This isn't a proper SemVer. The `patch` is for bugfixes. Shouldn't it be `1.1.0` in this case? Also, please don't bump a version in the PR.